Objects/typeobject.c

```diff
@@ -146,7 +146,9 @@ _PyType_CheckConsistency(PyTypeObject *type)
         return 1;
     }
 
-    CHECK(Py_REFCNT(type) >= 1);
+    if (!_PyObject_IsImmortal((PyObject *)type)) {
+        CHECK(Py_REFCNT(type) >= 1);
```

Comment:
I don't understand this change. Does it mean that refcount of immortal objects are negative and so Py_DECREF() on this type will call its deallocator?

Reply:
Hmm, I thought I had removed this.  It is a holdover from when I used a negative refcount to indicate immortal and no longer applies.  I'll fix it.